markdown/clientaccess/kerberos-userauth.html.md.erb

```diff
@@ -0,0 +1,459 @@
+    
+    ``` shell
+    root@kdc-server$ kadmin.local -q "xst -norandkey -k /etc/security/keytabs/hawq.service.keytab postgres/<master>@REALM.DOMAIN"
+    ```
+    
+    **Otherwise**, generate a new file for the keytab:
+
+    ``` shell
+    root@kdc-server$ kadmin.local -q "xst -norandkey -k hawq-krb5.keytab postgres/<master>@REALM.DOMAIN"
+    ```
+
+4. Use the `klist` command to view the key you just generated:
+
+    ``` shell
+    root@kdc-server$ klist -ket ./hawq-krb5.keytab
+    ```
+    
+    Or:
+    
+    ``` shell
+    root@kdc-server$ klist -ket /etc/security/keytabs/hawq.service.keytab
+    ```
+    
+    The `-ket` option lists the keytabs and encryption types in the identified key file.
+
+5. When you enable Kerberos user authentication for HAWQ, you must create a Kerberos principal for `gpadmin` or another HAWQ administrative user. Create a Kerberos principal for the HAWQ `gpadmin` administrative role, substituting your Kerberos realm. For example:
+
+    ``` shell
+    root@kdc-server$ kadmin.local -q "addprinc -pw changeme gpadmin@REALM.DOMAIN"
+    ```
+    
+    This `addprinc` command adds the principal `gpadmin` to the Kerberos KDC managing your \<realm\>. When you invoke `kadmin.local` as specified in the example above, `gpadmin` will be required to provide the password identified by the `-pw` option when authenticating. Alternatively, you can create a keytab file for the `gpadmin` principal and distribute the file to HAWQ client nodes.
+
+6. Copy the file in which you added the `postgres/<master>@<realm>` keytab to the HAWQ master node:
+
+    ``` shell
+    root@kdc-server$ scp ./hawq-krb5.keytab gpadmin@<master>:/home/gpadmin/
+    ```
+    
+    Or:
+    
+    ``` shell
+    root@kdc-server$ scp /etc/security/keytabs/hawq.service.keytab gpadmin@<master>:/etc/security/keytabs/hawq.service.keytab
+    ```
+
+## <a id="hawq_kerb_cfg_adkdc"></a>Step 1b: Configuring the HAWQ Principal using an AD KDC Server
+
+Perform the following procedure to configure a HAWQ Kerberos principal when you are using an AD KDC server.
+
+1. Log on to the Windows Active Directory Kerberos KDC server system as a user with administrator privileges.
+
+2. From the **Start** menu, select **Control Panel > Administrative Tools > Active Directory Users and Computers**. (If the **Active Directory Users and Computers** menu item is not available, the Active Directory service may not have been (correctly) installed.)
+
+    The **Active Directory Users and Computers** window displays.
+
+3. When you enable Kerberos user authentication for HAWQ, you must create a Kerberos principal for the `gpadmin` HAWQ administrative user. Use the left pane tree view to navigate to your Kerberos \<realm\> **Managed Service Accounts** folder, right-click, and select **New > User** to create a user with this name.
+
+    The **New Object - User** wizard displays.
+   
+4. Fill in the **New Object - User** fields:
+
+    **First name:**  gpadmin  
```

Comment:
I think we typically format these entries as 2-column tables.

Reply:
i can make this a table if you want.  i was using the convention we used when we documented ranger UI steps.  see step 1d in this link:

http://docs-hdb-review.cfapps.io/review/hawq/ranger/ranger-kerberos.html#kerb_rps2ra_proc

Reply:
No - problem, let's keep it as-is given that there's a precedent for it.